functests/1_performance/performance.go

```diff
@@ -1085,6 +1085,17 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Verify containers specifications", func() {
+
+		It("imagePullPolicy should be ifNotPresent", func() {
+			pod, err := pods.GetPerformanceOperatorPod()
+			Expect(err).ToNot(HaveOccurred())
+
+			// verify that the pod uses ifNotPresent as imagePullPolicy
```

Comment:
redundant comment, what is being checked is evident from the code. If anything, better reference here the BZ or the issue which this test is covering.